src/NuGet.Clients/NuGet.VisualStudio.Common/GlobalSuppressions.cs

```diff
@@ -54,3 +54,8 @@
 [assembly: SuppressMessage("Build", "CA1305:The behavior of 'string.Format(string, params object[])' could vary based on the current user's locale settings. Replace this call in 'PackageManagementFormat.PackageFormatSelectorLabel.get' with a call to 'string.Format(IFormatProvider, string, params object[])'.", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.VisualStudio.PackageManagementFormat.PackageFormatSelectorLabel")]
 [assembly: SuppressMessage("Build", "CA2227:Change 'ProjectNames' to be read-only by removing the property setter.", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.VisualStudio.PackageManagementFormat.ProjectNames")]
 [assembly: SuppressMessage("Build", "CA1822:Member OperationId does not access instance data and can be marked as static (Shared in VisualBasic)", Justification = "<Pending>", Scope = "member", Target = "~P:NuGet.VisualStudio.SolutionRestoreRequest.OperationId")]
+[assembly: SuppressMessage("Build", "CA1816:Change OutputConsoleLogger.Dispose() to call GC.SuppressFinalize(object). This will prevent derived types that introduce a finalizer from needing to re-implement 'IDisposable' to call it.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.Common.OutputConsoleLogger.Dispose")]
+[assembly: SuppressMessage("Build", "CA1063:Modify 'OutputConsoleLogger.Dispose' so that it calls Dispose(true), then calls GC.SuppressFinalize on the current object instance ('this' or 'Me' in Visual Basic), and then returns.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.Common.OutputConsoleLogger.Dispose")]
+[assembly: SuppressMessage("Build", "CA1062:In externally visible method 'void OutputConsoleLogger.Log(ILogMessage message)', validate parameter 'message' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.Common.OutputConsoleLogger.Log(NuGet.Common.ILogMessage)")]
+[assembly: SuppressMessage("Build", "CA1823:Unused field 'LogEntrySource'.", Justification = "<Pending>", Scope = "member", Target = "~F:NuGet.VisualStudio.Common.OutputConsoleLogger.LogEntrySource")]
+[assembly: SuppressMessage("Build", "CA1063:Provide an overridable implementation of Dispose(bool) on 'OutputConsoleLogger' or mark the type as sealed. A call to Dispose(false) should only clean up native resources. A call to Dispose(true) should clean up both managed and native resources.", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.VisualStudio.Common.OutputConsoleLogger")]
```

Comment:
How about fixing the class so that the suppression is not needed?

Reply:
time critical work for the build demo, not going to do this now...as there is much work left.